account-management-app/src/main/java/org/duracloud/account/app/controller/DuracloudMillForm.java

```diff
@@ -32,19 +29,29 @@
     private String auditLogSpaceId;
     @NotNull
     private String queueType;
+    private Boolean globalPropsAvailable;
```

Comment:
What about calling this `globalPropsRmqConfAvailable`? I know that's long, but I think it's more clear about what this variable is meant to tell you. As it is, it seems like it's telling you if there are any global properties available. Since it's tied to the RabbitMQ config, I'd like the name to reflect that.

Reply:
Renamed the variable and methods